tools/fast_nvcc/fast_nvcc.py

```diff
@@ -157,6 +157,13 @@ def warn_if_tmpdir_set(env):
         fast_nvcc_warn(url_vars)
 
 
+def contains_non_executable(commands):
+    for command in commands:
+        if command.startswith("--"):
```

Comment:
I don't quite understand what this means but I will try to play around with it a bit more to figure it out

Reply:
good point. I guess I should add comments on what things were adderssed in these situations. will follow up. thanks